Dockerfile

```diff
@@ -66,7 +66,7 @@ RUN wget https://bootstrap.pypa.io/get-pip.py && \
 
 # >> START Configure Tensorflow Build
 
-# Clone TensoFlow from Mozilla repo
+# Clone TensorFlow from Mozilla repo
```

Comment:
You need to re-base

Reply:
Josh could you rebase and push?

Reply:
broken rebase, so I made a new branch and PR... :(